### PR TITLE
'NSObject.hashValue' is not overridable; did you mean to override 'NSObject.hash'?

### PR DESCRIPTION
**Pull request by lukaswuerzburger**
Solves #191

Note: I used a newer CocoaPods version: 1.6.1. It used to be 1.5.3. But the actual current version is 1.7.3 though. Which one do you prefer?

There are some changes in to `project.pbxproj` files you might want to check.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -30,11 +30,11 @@ SPEC REPOS:
...
-  SwiftLint: bfa7ca7b4d170cfaf0d236ca3ffd969e88a2f002
+  Nimble: 051e3d8912d40138fa5591c78594f95fb172af37
+  OHHTTPStubs: 9cbce6364bec557cc3439aa6bb7514670d780881
+  Quick: f4f7f063c524394c73ed93ac70983c609805d481
+  SwiftLint: fed9c66336e41fc74dc48a73678380718f0c8b0e
 
 PODFILE CHECKSUM: 60fb0006e8002054214558b4c26542528124b72f
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.6.1
```
Comment (brototyp) [inline Podfile.lock]: I personally would prefer to use the latest Cocoapod version but I am fine with this here as well.

**Comment by brototyp** [comment]
The swiftlinting seem to fail. Do you mind trying a `bundle exec rake test` locally?

File: Podfile.lock (hunk 1 above)
Comment (lukaswuerzburger) [reply Podfile.lock]: I also prefer to use the latest version.

**Comment by lukaswuerzburger** [comment]
> The swiftlinting seem to fail. Do you mind trying a `bundle exec rake test` locally?

I'll have a look into this later today

**Comment by brototyp** [comment]
Hey @lukaswuerzburger, please have a look at #194. I fixed the build issue there. Aside of that, it should be mostly equal to your PR here.

**Pull request closed without merging**